src/sparsevector.jl

```diff
@@ -1468,7 +1469,7 @@ function _binarymap_mode_1!(f::Function, mx::Int, my::Int,
     return ir
 end
 
-function _binarymap_mode_2!(f::Function, mx::Int, my::Int,
+function _binarymap_mode_2!(f::Function, mx::Integer, my::Integer,
```

Comment:
but now `ix` will not have the same type as `mx`, i think the safe thing is just make every function take Int and then store the "right" type. `Int` seems to be fairly standard for indexing in julia IMHO.

Reply:
My suggestion would be to undo all changes after line 1395, unless we test for type stability.

Reply:
Ok, I undid changes after line 1395 and rebased my commit to keep history clean